src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_commands.c

```diff
@@ -231,37 +224,23 @@ char* wm_agent_upgrade_process_upgrade_custom_command(const int* agent_ids, wm_u
 
 char* wm_agent_upgrade_process_agent_result_command(const int* agent_ids, wm_upgrade_agent_status_task* task) {
     // Only one id of agent will reach at a time
-    int agent_id = agent_ids[0];
-
-    mtinfo(WM_AGENT_UPGRADE_LOGTAG, WM_UPGRADE_ACK_RECEIVED, agent_id, task->error_code, task->message);
-
-    // Send task update to task manager and bring back the response
-    cJSON *response = wm_agent_upgrade_send_single_task(WM_UPGRADE_AGENT_UPDATE_STATUS, agent_id, task->status);
-
-    if (wm_agent_upgrade_validate_task_status_message(response, NULL)) {
-        // If status update is successful, tell agent to erase results file
-        char *buffer = NULL;
-
-        os_calloc(OS_MAXSTR, sizeof(char), buffer);
-        snprintf(buffer, OS_MAXSTR, "%03d com clear_upgrade_result -1", agent_id);
-
-        char *agent_response = wm_agent_upgrade_send_command_to_agent(buffer, strlen(buffer)); 
-        char *data = NULL;
-        char *error = NULL;
-
-        if (wm_agent_upgrade_parse_agent_response(agent_response, &data, &error) == OS_SUCCESS) {
-            mtdebug1(WM_AGENT_UPGRADE_LOGTAG, WM_UPGRADE_UPGRADE_FILE_AGENT);
-        } else {
-            mterror(WM_AGENT_UPGRADE_LOGTAG, WM_UPGRADE_RESULT_FILE_ERROR, data);
-        }
+    int agent_id = 0;
+    int agent = 0;
+    cJSON* json_response = cJSON_CreateArray();
+    cJSON *json_task_module_request = cJSON_CreateArray();
```

Comment:
Free this JSON at the end of the function

Reply:
Done